.gitignore

```diff
@@ -28,3 +28,6 @@ test/native/junit.xml
 phpcs.xml
 phpunit.xml
 phpunit-watcher.yml
+
+# IDE
+.idea/
```

Comment:
Thanks for contributing, @mauteri. IDE and editor-related files should be ignored locally, maybe using a global gitignore file.

Reply:
Yes, we should remove this.

Reply:
Thanks, I'll remove it.